src/calendar/CalendarMonth.jsx

```diff
@@ -30,11 +24,39 @@ const CalendarMonth = React.createClass({
     onMonthChange: React.PropTypes.func,
     onYearChange: React.PropTypes.func,
     value: CustomPropTypes.momentOrMomentRange,
+    locale: React.PropTypes.string,
+  },
+
+  _setupLocale(locale) {
+    this.moment = moment();
+
+    if (locale !== 'en') {
+      require(`moment/locale/${locale}`);
```

Comment:
I don't believe this will work with bundling tools like Webpack or Browserify which rely on static `require` statements to figure out what dependencies need to be included in the bundle at compile time.